roachtest: django failed

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177076&tab=buildLog) on [release-20.1@8352fd645f4517ab0650ad637e22b20d9e524d53](https://github.com/cockroachdb/cockroach/commits/8352fd645f4517ab0650ad637e22b20d9e524d53):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.4-48-g8352fd6
		Tests run against django cockroach-3.0.x
		8029 Total Tests Run
		8028 tests passed
		1 test failed
		492 tests skipped
		20 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177076&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2190692&tab=buildLog) on [release-20.1@3d16e0d280a2aab638373ff563199dca0bb5a915](https://github.com/cockroachdb/cockroach/commits/3d16e0d280a2aab638373ff563199dca0bb5a915):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.4-52-g3d16e0d
		Tests run against django cockroach-3.0.x
		8030 Total Tests Run
		8028 tests passed
		2 tests failed
		491 tests skipped
		20 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_attribute_name_not_python_keyword (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2190692&tab=artifacts#/django)
Related:
- #52973 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52879 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008151325_v19.2.10](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008151325_v19.2.10) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52762 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2200626&tab=buildLog) on [release-20.1@18c2c6bb8079a123e5fa37d2e99211d6feefb29b](https://github.com/cockroachdb/cockroach/commits/18c2c6bb8079a123e5fa37d2e99211d6feefb29b):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.4-54-g18c2c6b
		Tests run against django cockroach-3.0.x
		8030 Total Tests Run
		8028 tests passed
		2 tests failed
		491 tests skipped
		20 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_updates_in_transaction (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2200626&tab=artifacts#/django)
Related:
- #53098 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008191705_v19.2.10](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008191705_v19.2.10) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52973 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

- #52762 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2214769&tab=buildLog) on [release-20.1@0df968b1e237ccd88cef89e851bcd52e90932280](https://github.com/cockroachdb/cockroach/commits/0df968b1e237ccd88cef89e851bcd52e90932280):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.4-65-g0df968b
		Tests run against django cockroach-3.0.x
		8029 Total Tests Run
		8028 tests passed
		1 test failed
		491 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2214769&tab=artifacts#/django)
Related:
- #52762 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2219151&tab=buildLog) on [release-20.1@5ccfb78ceeefd88a89ee4fad08ec415770c9a15a](https://github.com/cockroachdb/cockroach/commits/5ccfb78ceeefd88a89ee4fad08ec415770c9a15a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.4-73-g5ccfb78
		Tests run against django cockroach-3.0.x
		8029 Total Tests Run
		8027 tests passed
		2 tests failed
		491 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_updates_in_transaction (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2219151&tab=artifacts#/django)
Related:
- #53381 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008241848_v20.1.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008241848_v20.1.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52762 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/django.go
+++ b/pkg/cmd/roachtest/django.go
@@ -142,7 +142,7 @@ func registerDjango(r *testRegistry) {
 		if err := repeatRunE(
 			ctx, c, node, "install django's dependencies", `
 				cd /mnt/data1/django/tests &&
-				sudo pip3 install .. &&
+				sudo pip3 install -e .. &&
 				sudo pip3 install -r requirements/py3.txt &&
 				sudo pip3 install -r requirements/postgres.txt`,
 		); err != nil {

